UnitsNet/CustomCode/UnitValueAbbreviationLookup.cs

```diff
@@ -69,15 +69,21 @@ internal List<int> GetUnitsForAbbreviation(string abbreviation)
             return units.Distinct().ToList();
         }
 
-        internal void Add(int unit, string abbreviation)
+        internal void Add(int unit, string abbreviation, bool useAsDefault = false)
```

Comment:
If you add `setAsDefault` as I propose above, make sure these two names match. I slightly prefer `setAs` over `useAs`, but it's not a big deal.

Reply:
I can change the parameter name.

Reply:
✔️